js/book-carousel.js

```diff
@@ -0,0 +1,54 @@
+namespacer('bcpl');
+
+bcpl.bookCarousel = (($, constants) => {
+	const promises = [];
+	const slickSettings = {
+		infinite: true,
+		arrows: true,
+		prevArrow: '<a href="#"><i class="fa fa-chevron-left" aria-hidden="true" /></a>',
+		nextArrow: '<a href="#"><i class="fa fa-chevron-right" aria-hidden="true" /></a>',
+		slidesToShow: 3,
+		responsive: [{
+			breakpoint: constants.breakpoints.medium,
+			settings: {
+				slidesToShow: 2
+			}
+		}, {
+			breakpoint: constants.breakpoints.small,
+			settings: {
+				slidesToShow: 1
+			}
+		}]
+	};
+
+	const loadData = (carouselId) => {
+		return $.ajax(constants.shared.urls.bookCarousels.replace('CAROUSEL_ID', carouselId), {
+			dataType: 'jsonp'
+		}).then(data => {
+			const $items = $(data.Carousel_Str)
```

Comment:
I know this is kinda silly but we want to move the then to it's own function right?